### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by paulfalgout**
I also moved `getRegion` up next to `show`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (TerrenceLJones) [inline bower.json]: Ahh `bower` such a forgetful module loader.

Comment (paulfalgout) [inline bower.json]: heh.. yeah I'd lose it except that it is our 1 non-RWell contribution: https://github.com/RoundingWellOS/marionette.toolkit/pull/101

File: src/component.js [hunk 2]
```diff
@@ -135,6 +135,18 @@ const Component = Marionette.Object.extend({
   },
 
   /**
+   * Returns component region.
```
Comment (TerrenceLJones) [inline src/component.js]: Yea I figured you find a better placement for this.

**Comment by TerrenceLJones** [comment]
This looks good. 🚢

File: bower.json (hunk 1 above)
Comment (TerrenceLJones) [inline bower.json]: hah...yea...well we are being #inclusive!!

**Pull request merged**